Code/GraphMol/FindRings.cpp

```diff
@@ -1099,9 +1097,11 @@ int findSSSR(const ROMol &mol, VECT_INT_VECT &res) {
             << "WARNING: could not find number of expected rings. Switching to "
                "an approximate ring finding algorithm."
             << std::endl;
+        mol.getRingInfo()->reset();
         fastFindRings(mol);
         res.clear();
         res = mol.getRingInfo()->atomRings();
+        return rdcast<int>(res.size());
```

Comment:
By bailing out early here `FindRings::storeRingsInfo(mol, res)` is not executed. Is that intentional?

Reply:
yeah, there's no need to do that since `fastFindRings()` has already stored the rings it found.